website/docs/configuration/ort-yml.md

```diff
@@ -85,7 +85,7 @@ The path exclude above has the following effects:
 
 * All projects found below the `test-data` directory are marked as excluded.
 * License findings in files below the `test-data` directory are marked as excluded. This can be used in
-  [evaluator rules](../../tutorial/intro#6-running-the-evaluator) to for instance change the severity from error to
+  [evaluator rules](../../tutorial/intro.md#6-running-the-evaluator) to for instance change the severity from error to
```

Comment:
The link on the website works just fine if you test it here: https://oss-review-toolkit.org/ort/docs/configuration/ort-yml#excluding-paths

Because the tutorial is a separate instance of the docs plugin currently URL links have to be used, see the warning at the bottom of [this page](https://docusaurus.io/docs/markdown-features/links) or [this comment](https://github.com/facebook/docusaurus/issues/9117#issuecomment-1617569344).

Reply:
I just thought that we could merge the tutorial into the docs to avoid this issue by having only one instance of the docs plugin, what do you think?

Reply:
> only one instance of the docs plugin

Sorry, which docs plugin?

Anyway, you're the expert here, so whatever you say 😆 I was only addressing this as [the markdown-links check failed](https://github.com/oss-review-toolkit/ort/actions/runs/6668910751/job/18125541974#step:4:41).

Reply:
> Sorry, which docs plugin?

The docs plugin for docusaurus, we currently use two instances of the plugin, one for the docs and one for the tutorial. So links between those two must be URL links.

> Anyway, you're the expert here, so whatever you say 😆 I was only addressing this as the markdown-links check failed.

I will try to find the time to make that change in a separate PR, but until then the second commit should be removed as to my understanding it would actually break the link on the website. I wonder why the markdown lint check was failing, if this link is an issue it should also have failed before?

Reply:
> we currently use two instances of the plugin, one for the docs and one for the tutorial.

That sounds weird... and I wasn't aware of that. Thanks for pointing that out.

> I wonder why the markdown lint check was failing, if this link is an issue it should also have failed before?

Correct. Apparently some caching takes place until the particular file is touched in a PR...

Reply:
I think the ignore pattern in the `mlc_config.json` needs to be changed to `"^../../tutorial/intro.*$"` to silence the markdown link check issue.

Reply:
I gave it a try (with a slightly more specific pattern).